src/wallet/gtest/test_rpc_wallet.cpp

```diff
@@ -195,40 +195,40 @@ TEST(WalletRPCTests, RPCZsendmanyTaddrToSapling)
     CDataStream ss(ParseHex(hexTx), SER_NETWORK, PROTOCOL_VERSION);
     CTransaction tx;
     ss >> tx;
-    ASSERT_FALSE(tx.vShieldedOutput.empty());
+    ASSERT_FALSE(tx.GetSaplingOutputsCount() == 0);
```

Comment:
Consider setting `const auto& output0 = tx.GetSaplingOutputs()[0];` here and using it below. I know it's a test, but I think it's more readable.